assets/dev/js/admin/admin.js

```diff
@@ -285,6 +287,24 @@
 			this.openGetHelpInNewTab();
 
 			this.roleManager.init();
+
+			this.runLandingPagesHandler();
```

Comment:
Can you move the code to the module and run on a trigger?

Reply:
what module?